hubs-discord-moderator-guidelines.md

```diff
@@ -71,3 +71,15 @@ _(All moderators are free to propose changes)_
 * **Off topic messages in the general channel**
   - Reply to the user that the general channel isn’t the best place for their messages and direct them to the proper channel.
   - Share the message link in the #mods channel and let the other moderators know that you’ve taken care of it.
+
+## Meetup channel Hubs room.
+
+If someone misbehaves in the meetup channel's Hubs room or sends inappropriate messages in the chat to the meetup channel, i.e. they violate our community rules, take the following action:
+
+1. Join the meetup room (if needed).
+2. Find the problem user and kick them (you can kick them by opening the people menu, clicking on their name, and then selecting kick).
+3. Prevent the problem user from viewing the meetup channel via one of the following methods (depending on the severity of the offense):
```

Comment:
I had to read this twice to be sure this was on Discord; it would be helpful if point 3 lead with that.

It wouldn't hurt point 1 to explicitly say on Hubs.